telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/TelemetryParser.kt

```diff
@@ -100,6 +100,13 @@ object TelemetryParser {
                 it.types.plus(it2.types),
                 it.metrics.plus(it2.metrics)
             )
+        }.let {
+            // Allow read in files to overwrite default definitions. First one wins, so the extra
```

Comment:
This should probably be a part of the function comment, documenting the specific behavior of the generator with respect to args.

Reply:
Added